socorro/unittest/external/postgresql/test_server_status.py

```diff
@@ -2,14 +2,32 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import os
-import socorro
+import contextlib
+import tempfile
+
 from nose.tools import eq_
 
 from socorro.external.postgresql import server_status
 
 from unittestbase import PostgreSQLTestCase
 
+TEMP_DIR = tempfile.gettempdir()
```

Comment:
What the point of this? I don't see any other occurrences of ``TEMP_DIR`` in this file.